app/api/sessions.py

```diff
@@ -107,7 +107,7 @@ def query(self, view_kwargs):
             session_type = safe_query(self, SessionType, 'id', view_kwargs['session_type_id'], 'session_type_id')
             query_ = query_.join(SessionType).filter(SessionType.id == session_type.id)
         if view_kwargs.get('microlocation_id') is not None:
-            microlocation = safe_query(self, Track, 'id', view_kwargs['microlocation_id'], 'microlocation_id')
+            microlocation = safe_query(self, Microlocation, 'id', view_kwargs['microlocation_id'], 'microlocation_id')
```

Comment:
> We still have many endpoints left to add to documentation and to dredd tests

There will be unit tests for this stuff, documentation seems up to date and sufficient right now afaik, Only 2 apis are remaining in dredd testing.